### PR TITLE
Report file and line number on invalid bp position

### LINKED ISSUE #89: Invalid breakpoint position message should be descriptive
Currently: 'Breakpoint line number is invalid'
It should additionally specify which file and line number were invalid.

### PR DESCRIPTION
**Pull request by matthewloring**
Fixes #89

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots PTAL.

File: lib/v8debugapi.js [hunk 1]
```diff
@@ -267,7 +267,8 @@ module.exports.create = function(logger_, config_, fileStats_) {
     if (breakpoint.location.line >= fileStats[matchingScript].lines) {
       return setErrorStatusAndCallback(cb, breakpoint,
         StatusMessage.BREAKPOINT_SOURCE_LOCATION,
-        messages.INVALID_LINE_NUMBER);
+        messages.INVALID_LINE_NUMBER + matchingScript + ':' +
+          breakpoint.location.line);
```
Comment (ofrobots) [inline lib/v8debugapi.js]: You should also indicate the number of lines that `matchingScript` has.

Maybe we should suggest that perhaps their source doesn't match the deployment image?

Comment (matthewloring) [inline lib/v8debugapi.js]: Done.

**Comment by ofrobots** [comment]
LGTM.

**Pull request merged**